Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm

```diff
@@ -1112,6 +1112,18 @@ - (void)URLSession:(NSURLSession *)session dataTask:(NSURLSessionDataTask *)data
         auto privateRelayed = metrics._privacyStance == nw_connection_privacy_stance_direct
             || metrics._privacyStance == nw_connection_privacy_stance_not_eligible
             ? PrivateRelayed::No : PrivateRelayed::Yes;
+        String proxyName = "default"_s;
+        if (metrics._establishmentReport) {
+            if (auto endpoint = nw_establishment_report_copy_proxy_endpoint(metrics._establishmentReport)) {
+                if (const char * hostname = nw_endpoint_get_hostname(endpoint))
+                    proxyName = String::fromUTF8(unsafeMakeSpan(hostname, strlen(hostname)));
+                else
+                    LOG(NetworkSession, "%llu didReceiveResponse hostname is null.", taskIdentifier);
```

Comment:
These will happen a lot when private relay is off.  Also, I use the network session logs for debugging a lot and they are already pretty noisy.  Could we remove these logs?

Reply:
I can remove them for now. I will see if there is way to find out if private relay is enabled and conditionally add them.